vehicle/OVMS.V3/components/vehicle/vehicle.cpp

```diff
@@ -82,6 +82,13 @@ OvmsVehicleFactory::OvmsVehicleFactory()
   MyCommandApp.RegisterCommand("unlock","Unlock vehicle",vehicle_unlock,"<pin>",1,1);
   MyCommandApp.RegisterCommand("valet","Activate valet mode",vehicle_valet,"<pin>",1,1);
   MyCommandApp.RegisterCommand("unvalet","Deactivate valet mode",vehicle_unvalet,"<pin>",1,1);
+  OvmsCommand *cmd_aux = cmd_vehicle->RegisterCommand("aux", "Aux battery", vehicle_aux);
```

Comment:
Not exactly sure what this aux command does. 
Flagging it to check later.

Reply:
Seems fine.

Reply:
On a second note. This could actually be quite useful to track the battey status for our cars.